README.md

```diff
@@ -2,15 +2,15 @@ This is a user-space file system for interacting with [Google Cloud
 Storage][gcs]. It is not yet ready for public consumption, so please do not use
 it for anything of importance in its current state.
 
-[gcs]: https://cloud.google.com/storage/
+[gcs]: https://cloud.google.com/storage/docs
 
 
 # Installing
 
-See [installing.md][] for full installation instructions. The summary is that if
+See [installing.md][installing.md] for full installation instructions. The summary is that if
```

Comment:
My bad. I clicked it and thought it landed on the same page.  But I see it works fine.